### ISSUE TITLE
Can I change FPS of camera in Lincoln2017mkz?

### ISSUE THREAD

**Issue by kdheejb7** [lgsvl/simulator#1833]
Hello, I'm a user of lgsvl.

I'm using the lgsvl with the Apollo system, and I'm testing with vehicle type Lincoln2017mkz and map type Borregas Ave now.

I have confirmed that the Lincoln2017mkz's camera has 15 fps. 
![image](https://user-images.githubusercontent.com/39160667/138652885-f4829f5c-ada5-4bce-847a-4f6ee8f32a99.png)

But I want to change the fps to higher.
What part of which file do I need to edit to change FPS of the camera in Lincoln2017mkz? 

Thank you

**Comment by mslavescu** [comment]
@kdheejb7  
Did you also set the TargetFPS to 50?

What are the specs of the machine where you run SVL?
If you still see lower FPS after you set TargetFPS to 50, that normally happens because the machine (CPU/GPU) you use is not fast enough to reach the TargetFPS.

**Comment by EricBoiseLGSVL** [comment]
Yes, @mslavescu is right.  What GPU are you using?  What other applications are you running with simulator?  Are you running Apollo or Autoware on the same machine?  More details will help.  You can also check simulator fps by opening the info panel in the UI.
![image](https://user-images.githubusercontent.com/45106230/139753523-5461695c-5cf1-407a-9a98-d7aea4d5ec72.png)
